docker-compose.yml

```diff
@@ -26,7 +26,7 @@ services:
       - ./docker/database.sh:/docker-entrypoint-initdb.d/init.sh
       - ./sql:/wacadb
   msgbroker:
-    image: rabbitmq:3.10-management-alpine
+    image: rabbitmq:3.8-management-alpine
```

Comment:
Bleh, my downgrade here is because we move from 3.7 -> 3.8 with the.migration from buster to bullseye.

I'm more inclined to have docker match production for the time being to help avoid compatibility issues.

Reply:
Fair enough! Just please go ahead and merge #768 after this one then! :D